lnschema_core/_core.py

```diff
@@ -451,6 +451,34 @@ def path(self) -> Union[Path, CloudPath]:
         """Path on storage."""
         return filepath_from_file(self)
 
+    def replace(self, data: Union[Path, str, pd.DataFrame, ad.AnnData], source: Optional[Run] = None, format: Optional[str] = None):
+        """Replace data."""
+        from lamindb._file import get_path_size_hash, serialize
```

Comment:
I'm wondering whether one should rather use this here: https://github.com/laminlabs/lnschema-core/blob/5e21b4c9651f7994430b67a208606ba166a30aad/lnschema_core/_core.py#L525

Reply:
First i used this, but decided not to to avoid redundant `select` for `Storage`.